app/static/js/userProfile.js

```diff
@@ -61,12 +62,38 @@ $(document).ready(function(){
   });
 
   $("#addNoteButton").click(function() {
+    $("#noteDropdown").show()
+    $("#bonnerStatement").hide()
+    $("#VisibilityLabel").show()
+    $("#bonnerInput").prop("checked", false);
     $("#noteModal").modal("toggle");
+  
   });
 
+  $("#addVisibility").click(function() { 
+    var bonnerChecked = $("input[name='bonner']:checked").val()
+    
+  if (bonnerChecked == 'on') {
+    $("#noteDropdown").hide()
+    $("#bonnerStatement").show()
+    $("#VisibilityLabel").hide()
```

Comment:
We should be contestant with our naming, the v in `VisibilityLabel` should be lowercase.